src/dump.c

```diff
@@ -784,6 +784,11 @@ static void jl_serialize_value_(jl_serializer_state *s, jl_value_t *v, int as_li
     else if (jl_typeis(v, jl_task_type)) {
         jl_error("Task cannot be serialized");
     }
+#ifdef JULIA_ENABLE_PARTR
+    else if (jl_typeis(v, jl_condition_type)) {
+        jl_error("Condition cannot be serialized");
+    }
```

Comment:
This is not really necessary; might as well allow serializing a Condition as long as it doesn't point to any non-serializable tasks.

Reply:
Right above this it says "Task cannot be serialized"?

Reply:
Yes, if there are tasks inside you'll get that error, but we need to be able to save empty Conditions, for example in the global semaphore in threadcall.jl.